templates/category.twig

```diff
@@ -0,0 +1,43 @@
+{% extends "base.twig" %}
+
+{% block content %}
+
+	<div class="clearfix"></div>
+	<div class="skewed-overlay"></div>
+
+	<header class="page-header">
+		<div class="container">
+			<h1 class="page-header-title">{{ page_type.name }}</h1>
+			<div class="page-header-content">
+				<p>{{ page_type.description }}</p>
+			</div>
+		</div>
+	</header>
+
+	<div class="page-template">
+		<h3>{{ __( 'Results', 'planet4-master-theme' ) }}</h3>
+
+		<div class="row">
+			<div class="col-lg-8 multiple-search-result">
+				<ul class="list-unstyled">
+					{% for post in posts %}
+						{% include 'tease-page-type.twig' %}
```

Comment:
this fails because there are some specific variables for page type used in the template
https://github.com/greenpeace/planet4-master-theme/blob/develop/templates/tease-page-type.twig#L7